admin/src/js/controllers/edit-user.js

```diff
@@ -408,22 +415,22 @@ angular
           return CreateUser.createSingleUser(updates);
         })
         .then(() => {
-          $scope.setFinished();
+          $scope.status = 'Saved';
           // TODO: change this from a broadcast to a changes watcher
           //       https://github.com/medic/medic/issues/4094
           $rootScope.$broadcast(
             'UsersUpdated',
             $scope.editUserModel.id
           );
-          $uibModalInstance.close();
+          $state.go('users');
         })
         .catch(err => {
           if (err && err.data && err.data.error && err.data.error.translationKey) {
             $translate(err.data.error.translationKey, err.data.error.translationParams).then(function(value) {
-              $scope.setError(err, value);
+              $log.error(value, err);
```

Comment:
`$scope.setError` used to show the error in the footer of the modal so the user could see what went wrong. It looks like this is just putting it in the console which most users won't check. Let's make sure the user gets some feedback.